lib/ret_web/router.ex

```diff
@@ -197,6 +197,16 @@ defmodule RetWeb.Router do
     forward "/", Absinthe.Plug, json_codec: Jason, schema: RetWeb.Schema
   end
 
+  scope "/api-internal", RetWeb do
+    pipe_through(
+      [:portal_header_auth, :secure_headers, :parsed_body, :api] ++ if(Mix.env() == :prod, do: [:ssl_only], else: [])
+    )
+
+    scope "/v1", as: :api_v1 do
```

Comment:
I forget what `as: :api_v1` does here but we depending on what it does we maybe want something like `as: api_internal_v1`. My guess is its for generating utility functions.